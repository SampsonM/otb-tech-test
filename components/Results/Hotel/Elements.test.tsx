import React from "react"
import { render } from "@testing-library/react"
import { Container } from './Elements'

describe("Elements", () => {
	test("Container fragment matches the snapshot", () => {
		const { asFragment } = render(<Container />)
		expect(asFragment()).toMatchSnapshot()
	})
})
