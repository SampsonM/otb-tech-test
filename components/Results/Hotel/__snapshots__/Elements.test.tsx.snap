// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Elements Container fragment matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="Elements__Container-otb__sc-eb6dccc9-0"
  />
</DocumentFragment>
`;
